### DISCUSSION TITLE
How to batch delete Charges?

### DISCUSSION THREAD

**Discussion by martin0607** [teslamate-org/teslamate#2334] [Q&A]
A huge jump from 81 to 2140, what happened between these to numbers? How can I delete these invisible charges in one time?
![2022-01-18 200039](https://user-images.githubusercontent.com/32560181/149933707-3c3e980b-f26c-4244-ba80-1ce33a1b121d.jpg)
![2022-01-18 195544](https://user-images.githubusercontent.com/32560181/149933072-2be4633d-e529-4af1-80d1-272b45a6c692.jpg)
![2022-01-18 195725](https://user-images.githubusercontent.com/32560181/149933410-33670408-b4d9-401b-ba7a-6fba993c674e.jpg)

**Comment by cwanja** [comment]
Look at the postgresql delete command. You would need to give it a range: 
`DELETE FROM table WHERE field > value1 AND field < value2;`
